src/code/views/share-dialog-view.coffee

```diff
@@ -170,40 +170,40 @@ module.exports = React.createClass
           if sharing
             (div {},
               (div {className: 'share-status'},
-                "Shared view is ", (strong {}, "enabled")
-                (a {href: '#', onClick: @toggleShare}, 'Stop sharing')
+                tr "~SHARE_DIALOG.SHARE_STATE", {state: tr "~SHARE_DIALOG.SHARE_STATE_ENABLED"}
```

Comment:
I did remove it intentionally, but I am rethinking. The phrasing is as a single clause sentence and therefore the replaceable parameter ('enabled' or 'disabled') might be positioned at a different point for different translations. There isn't a way to incorporate internal styling in replaceable text with the `tr` method as far as I understand. So it was an option of keeping the sentence structure or keeping the styling.

@ddamelin, do you have a viewpoint on this? We could have "Shared view is enabled" and "Shared view is disabled" with no styling, or something like "Shared view: _enabled_" and "Shared view: _disabled_" with the styling. That is we could have structured text so that the position of the replaceable parameter is enforced. Do you have a preference?

Reply:
I think restructuring to Shared view: enabled is likely better because enabled/disabled is currently appended to the phrase Shared view is. So I'm guessing in the translation database we would have "Shared view is" and "enabled" and "disabled" all as separate translatable strings. However, I think your argument is that this phrase might be reordered based on the language. In that case it would be easier to use the "Shared view:" style. Otherwise we would need to put in the entire phrase to be translated together and change how we display this string (making it one phrase).